### ISSUE TITLE
Set up Responsive Images

### ISSUE THREAD

**Issue by jensimmons** [jensimmons/thewebahead#7]
Drupal can handle responsive images automagically. There are some new modules for that. I think I've dropped them into the repo already. Nothing's been setup. First step: figure out how the whole thing works. And set it up.

I've not setup any custom image styles yet for this site. I've just been quickly & lazily, randomly using image styles that install with core — with a plan to later define and implement the real sizes needed. I'm not sure if responsive images can be setup without having Real image styles. Or if the image styles need to be figured out and defined first. 

Let's discuss in this thread what need to happen as any of us figures out these bits.

**Comment by jensimmons** [comment]
Ooo, I just realized I hadn't dropped the modules yet. Just did: Picture and Breakpoint. 

Tutorial here: http://fourword.fourkitchens.com/article/responsive-images-drupal-implementation and likely elsewhere too.

**Comment by jensimmons** [comment]
Hi @proko 
I thought I'd respond here instead of email.
I'd love to have you work on this. Please ask any questions you have here. I haven't figured out yet what the image styles should be. I created "temp circle" to be just that, a temporary circle experiment. Mostly images show up at random sizes. Once I've started designing layouts, I'll know what I want and start defining real styles. And responsive styles, breakpoints, etc.
Since I haven't used the picture element with Drupal yet, I'm not sure whether there's anything that can be setup yet — or if we should wait until the images are better understood. Feel free to poke around, in any case.

**Comment by jensimmons** [comment]
Ok, I just spent three days on this. My conclusion: Drupal's picture & breakpoint module solution is not ready yet. It seems over engineered, and seems to solve the wrong problem. I want to use srcset, in a simple manner, to swap out 1x, 2x and 3x images for different resolution screens. Breakpoint plus picture doesn't do that. Those modules are solving a need to swamp out different-sized, but otherwise identical images at different screens sizes, to especially create "responsive images" without writing any CSS to resize the image. AKA, swap a 100px square with a 200px square and a 400px square so you don't have to write any CSS to take the one square and shrink it down / in a situation where the images are not fluid. 

So I simply used bigger images, and scaled them down with CSS — basically delivering 2x images to every device. Which sucks. But after 3 days of working on one bit of the site, it was the best I could do. Meanwhile, I did get all the images to be smaller by redoing their image styles (Why was I using png? The files are much smaller now as jpegs.)
